packages/apps/human-app/frontend/src/shared/components/ui/page-card/page-card.tsx

```diff
@@ -95,34 +62,35 @@ export function PageCard({
     },
   };
 
-  useEffect(() => {
-    if (withLayoutBackground && !isDarkMode) {
-      setGrayBackground();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps -- call this effect once
-  }, []);
-
-  const goBack = (pathOrCallback: ButtonsProps) => {
-    if (pathOrCallback instanceof Function) {
-      pathOrCallback();
+  const goBack = (navigationTarget: NavigationTarget) => {
+    if (navigationTarget instanceof Function) {
+      navigationTarget();
       return;
     }
-    if (typeof pathOrCallback === 'string') {
-      navigate(pathOrCallback);
+    if (typeof navigationTarget === 'string') {
+      navigate(navigationTarget);
       return;
     }
-    navigate(pathOrCallback);
+    navigate(navigationTarget);
```

Comment:
Looks like we don't need last line in this fn
```suggestion
    if (navigationTarget instanceof Function) {
      navigationTarget();
    } else {
      navigate(navigationTarget);
    }
```

Reply:
Actually I cannot do that; `navigate` has multiple definitions and eslint does not let me do that:

```
    (to: To, options?: NavigateOptions): void;
    (delta: number): void;
}
``` 
where 

```
export type To = string | Partial<Path>;
```

So we have to leave it as is.

Reply:
That seems reasonable from `useNavigate` developers.

From what I see, we use `-1` only as default value for `backNavigation`, so IMU we can get rid of it in `NavigationTarget` type, leaving it as `string | (() => void);`

Then we will be able to change the code to something like:
```
const followNavigationTarget = (navigationTarget: NavigationTarget) => {
    if (navigationTarget instanceof Function) {
      navigationTarget();
    } else if (typeof navigationTarget === 'string') {
      navigate(navigationTarget);
   } else {
      navigate(-1);
   }
```

Looks cleaner, wdyt?

Reply:
@mpblocky 
That seems reasonable from `useNavigate` developers.

From what I see, we use `-1` only as default value for `backNavigation`, so IMU we can get rid of it in `NavigationTarget` type, leaving it as `string | (() => void);`

Then we will be able to change the code to something like:
```
const followNavigationTarget = (navigationTarget: NavigationTarget) => {
    if (navigationTarget instanceof Function) {
      navigationTarget();
    } else if (typeof navigationTarget === 'string') {
      navigate(navigationTarget);
   } else {
      navigate(-1);
   }
```

Looks cleaner, wdyt?

Reply:
It looks pretty similar and we are taking away to go back one page. I can do that, but are You sure You want this change?

Reply:
I don't see a case when we might want to pass `-2` or different numeric value. IMU you either want a default "go back" behavior (i.e. to exact previous route, which is `-1`), or you specific some specific one with exact path or cb.

If the above is correct - then yes, I prefer to change the implementation to something explicit that won't cause confusion with `navigate` typings again :)

Reply:
@mpblocky I don't see a case when we might want to pass `-2` or different numeric value. IMU you either want a default "go back" behavior (i.e. to exact previous route, which is `-1`), or you specific some specific one with exact path or cb.

If the above is correct - then yes, I prefer to change the implementation to something explicit that won't cause confusion with `navigate` typings again :)

Reply:
Sounds good :)